api/src/main/java/com/velocitypowered/api/event/command/ProxyCommandExecuteEvent.java

```diff
@@ -8,17 +8,21 @@
 package com.velocitypowered.api.event.command;
 
 import com.google.common.base.Preconditions;
+import com.velocitypowered.api.command.CommandResult;
 import com.velocitypowered.api.command.CommandSource;
 
 /**
  * This event is fired when velocity executed a command. This event is called after the event
  * is handled. It's not fired for forwarded commands or commands with syntax errors.
+ *
+ * <p>Commands can be cancelled or forced to backend servers in {@link CommandExecuteEvent}
```

Comment:
If by _forced_ we mean _forwarded_, then the new event is expected to be fired, right? This would correspond to the `CommandResult.FORWARDED` value.

I agree that we shouldn't fire the new event if the execution is outright cancelled in `CommandExecuteEvent`.